authentication.md

```diff
@@ -139,3 +139,28 @@ Status codes:
 * 400 Bad Request - if the X-On-Behalf-Of header doesn't contain a valid EPerson UUID
 * 403 Forbidden - if you are not authorized to act on behalf of the given user
 * Any status code of the functionality being used
+
+
+## Request download token
+
+** POST /api/authn/download-token **
```

Comment:
I'm not clear what data should be POSTed to retrieve a download-token.  Could we provide some `curl` examples or similar for the expected POST command (similar to how we've done for the POST `/api/authn/login` command)?  For example, is it just the existing Authorization header (JWT) that needs posting, or do we also need info about which bitstream is being downloaded?

Reply:
I added a commit with an example, you only need to provide your existing JWT authorization header so you can get a short lived one for downloads.

Reply:
what about `/api/authn/shortlivedtokens` as endpoint? we have agreed to avoid the use of - and I would prefer to don't hardcode the use case of download in the name as it could be used in other scenario in future (maybe)

Reply:
Good idea, done !